nextcord/client.py

```diff
@@ -2023,33 +2039,171 @@ def get_application_command(self, command_id: int) -> Optional[BaseApplicationCo
         """
         return self._connection.get_application_command(command_id)
 
-    def get_application_command_from_signature(
-        self, name: str, cmd_type: Union[int, ApplicationCommandType], guild_id: Optional[int]
-    ) -> Optional[BaseApplicationCommand]:
+    def __actual_get_application_command_from_signature(
+        self,
+        *,
+        type: Union[int, ApplicationCommandType],
+        qualified_name: str,
+        guild_id: Optional[int],
+        search_locales: bool = False,
+    ) -> Optional[Union[BaseApplicationCommand, SlashApplicationSubcommand]]:
         """Gets a locally stored application command object that matches the given signature.
 
+        .. versionadded:: 2.0
+
+        .. versionchanged:: 2.3
+            it's no longer possible to pass ``None`` to the ``name`` parameter.
+
+        .. versionchanged:: 2.3
+            Renamed the ``name`` parameter to ``qualified_name``.
+
+        .. versionchanged:: 2.3
+            Renamed the ``cmd_type`` parameter to ``type``.
+
+        .. versionchanged:: 2.3
+            Added the ``search_locales`` keyword argument.
+
+        .. versionchanged:: 2.3
+            Group/subcommand commands can now be retrieved with this method.
+
+        .. versionchanged:: 2.3
+            Changed the signature from ``(qualified_name, type, guild_id, *, search_locales)`` to ``(*, type, qualified_name, guild_id, search_locales)``.
```

Comment:
This part is redundant

Reply:
How so? I think it's necessary to note the kwargs only change.

Reply:
Well the specific line I left this comment on seems unnecessary because the next line pretty much sums it up.